dotnet/samples/KernelSyntaxExamples/Example66_FunctionCallingStepwisePlanner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Planners;
+using Plugins;
+using RepoUtils;
+
+// ReSharper disable once InconsistentNaming
+public static class Example66_FunctionCallingStepwisePlanner
+{
+    internal static string? ChatModelOverride = null; //"gpt-35-turbo-0613"
+
+    public static async Task RunAsync()
+    {
+        string[] questions = new string[]
+        {
+            "Write a limerick and send it via email to John"
+        };
+
+        var kernel = InitializeKernel();
+
+        var config = new FunctionCallingStepwisePlannerConfig
```

Comment:
nit: without the `{}`
```suggestion
        var config = new FunctionCallingStepwisePlannerConfig();
```